### PR TITLE
Fixerupper

### PR DESCRIPTION
**Pull request by RogeredBacon**
I basically just did what you suggested in #133 and #130.

### CODE REVIEW

File: react-backend/client/src/components/visitors/main.js [hunk 1]
```diff
@@ -178,7 +181,8 @@ class Main extends Component {
 
               {/*This is the print layout of the QRcode*/}
               <div className="visible-printer qr-code-to-print">
-                <div className="dashed">
+                <div className="dashed">this.setError([errorMessages.NO_INPUT_ERROR]);
```
Comment (finnhodgkin) [inline react-backend/client/src/components/visitors/main.js]: This looks like an accidental middle click... Happens to me pretty regularly.

File: react-backend/client/src/components/errors.js [hunk 2]
```diff
@@ -64,5 +64,12 @@ export default {
     </span>
   ),
 
-  DETAILS_ERROR: <span>The email address or password is incorrect. Please try again.</span>,
+  DETAILS_ERROR: (
+  <span>The email address or password is incorrect. Please try again.</span>
+  ),
+
+  UNKOWN_ERROR: (
+    <span>Something weird has happened :S. <br />
```
Comment (finnhodgkin) [inline react-backend/client/src/components/errors.js]: I'd get rid of the ':S' unless the PO wants super friendly error message.

File: react-backend/client/src/components/errors.js [hunk 3]
```diff
@@ -64,5 +64,12 @@ export default {
...
+    Please try again or contact an system administrator if the problem persists.</span>
```
Comment (finnhodgkin) [inline react-backend/client/src/components/errors.js]: 'a system' rather than 'an'

**Comment by RogeredBacon** [comment]
Okie dokie, I've fixed the typos

**Comment by azayneeva** [comment]
I agree, looks good to me

**Pull request merged**